chain/block.go

```diff
@@ -819,6 +825,7 @@ func UnmarshalBlock(raw []byte, parser Parser) (*StatefulBlock, error) {
 	p.UnpackID(false, &b.StateRoot)
 	b.UnitsConsumed = p.UnpackUint64(false)
 	b.WarpResults = set.Bits64(p.UnpackUint64(false))
+	b.size = len(raw)
```

Comment:
Can we also set size after `Marshal`? Or, add a comment indicating it is only populated if `Unmarshaled`?

Reply:
If you add to `Marshal`, you can get the `Len` of the packer and can just use that.

Reply:
done